src/_guides/language/evolution.md

```diff
@@ -4,173 +4,175 @@ short-title: Language evolution
 
-Dart 2.5 didn't add any features to the Dart language, but it did add
-support for [calling native C code][] from Dart code
-using a new **core library, `dart:ffi`.**
+* [Enhanced enums][]
+* [Super parameters][]
+* [Named parameters][]
 
-### Dart 2.6
+[Enhanced enums]: /language/enum#declaring-enhanced-enums
+[Super parameters]: /language/constructors#super-parameters
+[Named parameters]: /language/functions#named-parameters
 
-Dart 2.6 didn't add any features to the Dart language, but it did add a
-**new tool, `dart2native`,** for compiling Dart code to
-native executables.
-This functionality has since been folded into the [`dart compile`][] command.
+### Dart 2.16
+_Released 3 February 2022_
+| [Dart 2.16 announcement](https://medium.com/dartlang/dart-2-15-7e7a598e508a)
 
-### Dart 2.7
+Dart 2.16 added no new features to the Dart language, but fixed one
+security vulnerability and made two breaking changes.
+It did expand the Dart tools.
 
-Dart 2.7 added support for **[extension methods][]**,
-enabling you to add functionality to any type—even types you don’t control—with
-the brevity and auto-complete experience of regular method calls.
-Because the tech preview for this feature was in 2.6,
-you can use extension methods without warnings if you
-specify 2.6.0 or a later release as the lower SDK constraint.
+### Dart 2.15
+_Released 8 December 2021_
+| [Dart 2.15 announcement](https://medium.com/dartlang/dart-2-15-7e7a598e508a)
 
-The following example extends the `String` class from `dart:core` with a new
-`parseInt()` method:
+Dart 2.15 improved support for function pointers, known as _tear-offs._
+In particular, constructor tear-offs are now supported.
 
-```dart
-extension ParseNumbers on String {
-  int parseInt() {
-    return int.parse(this);
-  }
-}
+### Dart 2.14
+_Released 8 September 2021_
+| [Dart 2.14 announcement](https://medium.com/dartlang/announcing-dart-2-14-b48b9bb2fb67)
 
-void main() {
-  int i = '42'.parseInt();
-  print(i);
-}
-```
+Dart 2.14 added the unsigned shift (or _triple-shift_) operator (`>>>`).
+This new operator works like `>>`,
+except that it always fills the most significant bits with zeros.
 
-### Dart 2.8
+Dart 2.14 removed some restrictions on type arguments.
+You can pass type arguments to annotations and use a generic function
+type as a type argument.
+As of Dart 2.14, you can write the following code:
 
-Dart 2.8 didn't add any features to the Dart language, but it did
-contain a number of preparatory [breaking changes][2.8 breaking changes] to ensure great
-nullability-related usability and performance in the upcoming
-[null safety][] feature.
+```dart
+@TypeHelper<int>(42, "The meaning")
+late List<T Function<T>(T)> idFunctions;
+var callback = [<T>(T value) => value];
+late S Function<S extends T Function<T>(T)>(S) f;
+```
 
-It also contained a faster pub tool, and a new [pub outdated][] command.
+To learn more about these operators, check out [bitwise and shift operator][].
```

Comment:
Sorry I missed this one before I submitted my review, but the text preceding this is not about the operator. Perhaps move this to right after the discussion of the operator.